main.js

```diff
@@ -0,0 +1,243 @@
+  },
+  {
+    projectID: 5,
+    name: 'Data Dashboard Healthcare',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    featuredImage: 'images/firstprojectbig.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    linkLiveVersion: 'https://www.google.com',
+    linkSource: 'https://www.github.com',
+    modalDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  },
+  {
+    projectID: 6,
+    name: 'Website Protfolio',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    featuredImage: 'images/firstprojectbig.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    linkLiveVersion: 'https://www.google.com',
+    linkSource: 'https://www.github.com',
+    modalDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  },
+];
+
+const dinamicProjects = document.querySelector('#dinamicProject');
+//  CARDS BEING GENERATED
+for (let index = 1; index < projects.length; index += 1) {
+  //  projects.forEach( (project, index)=> {
+  const project = projects[index];
+  //  Div projects
+  const divProject = document.createElement('div');
+  divProject.setAttribute('class', 'projects');
+  //  See project button
+  const btnSeeProject = document.createElement('button');
+  const btnText = document.createTextNode('See project');
+  btnSeeProject.setAttribute('class', 'buttons neonText2');
+  //  btnSeeProject.setAttribute("id", `btnSeeProject${project.projectID}`);
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove this commented code, they make your codebase not clean and professional.

Reply:
✔️